config/routes.rb

```diff
@@ -3,5 +3,8 @@
     root "static_pages#home"
     get "/home", to: "static_pages#home"
     get "/help", to: "static_pages#help"
+    get "/signup", to: "users#new"
+    post "/signup", to: "users#create"
+    resources :users
```

Comment:
cần giới hạn lại các action dùng nha e, ví dụ
```suggestion
    resources :users, only: %i(new create show)
```

Reply:
da, để em sửa lại